From 4a93c6c373cb4ace72be2bc7d12981c43ae8b1bc Mon Sep 17 00:00:00 2001
From: Manh Hoang <manh.hoang.uh@bp.renesas.com>
Date: Thu, 7 May 2026 23:56:50 +0700
Subject: [PATCH] ADSP-remove-HDMI-support-from-rcar-sound

This patch will remove HDMI support form rcar sound,
in case we use ADSP

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Manh Hoang <manh.hoang.uh@bp.renesas.com>
Signed-off-by: Loc Nguyen <loc.nguyen.wt@renesas.com>


---
 arch/arm64/boot/dts/renesas/r8a779m1-salvator-x.dts | 6 ------
 arch/arm64/boot/dts/renesas/r8a779m5-salvator-x.dts | 5 -----
 2 files changed, 11 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a779m1-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a779m1-salvator-x.dts
index 4c8e7ad8e8c5..ef20f784a75d 100644
--- a/arch/arm64/boot/dts/renesas/r8a779m1-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a779m1-salvator-x.dts
@@ -183,12 +183,6 @@ &sata {
 	status = "okay";
 };
 
-&sound_card {
-	dais = <&rsnd_port0	/* ak4613 */
-		&rsnd_port1	/* HDMI0  */
-		&rsnd_port2>;	/* HDMI1  */
-};
-
 &usb2_phy2 {
 	pinctrl-0 = <&usb2_pins>;
 	pinctrl-names = "default";
diff --git a/arch/arm64/boot/dts/renesas/r8a779m5-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a779m5-salvator-x.dts
index d99e2f5fa86a..a09cdf81cae1 100644
--- a/arch/arm64/boot/dts/renesas/r8a779m5-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a779m5-salvator-x.dts
@@ -134,11 +134,6 @@ rsnd_endpoint1: endpoint {
 	};
 };
 
-&sound_card {
-	dais = <&rsnd_port0	/* ak4613 */
-		&rsnd_port1>;	/* HDMI0  */
-};
-
 &vspb {
 	status = "okay";
 };
-- 
2.34.1

